ctl/log/log.go

```diff
@@ -26,20 +26,34 @@ import (
 	"strings"
 
 	"github.com/spf13/cobra"
+	"istio.io/istio/pkg/kube"
+)
+
+const (
+	KmeshNamespace = "kmesh-system"
+	KmeshAdminPort = 15200
 
-	"kmesh.net/kmesh/pkg/status"
+	patternLoggers = "/debug/loggers"
 )
 
+type LoggerInfo struct {
+	Name  string `json:"name,omitempty"`
+	Level string `json:"level,omitempty"`
+}
+
 func NewCmd() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "log",
 		Short: "Get or set kmesh-daemon's logger level",
-		Example: `Set default logger's level as "debug":
-		kmesh-daemon log --set default:debug
+		Example: `# Set default logger's level as "debug":
+kmeshctl log <kmesh-daemon-pod> --set default:debug
```

Comment:
Do I have to specify the kmesh pod name? 
nit：`kmeshctl bpf-log --set debug` is clear and simple?

Reply:
do i need to specify namespace?

Reply:
> Do I have to specify the kmesh pod name? nit：`kmeshctl bpf-log --set debug` is clear and simple?

Yes, there are many Kmesh daemon pods in the cluster.

It should be able to tune more loggers, not just bpf.

Reply:
> do i need to specify namespace?

Not necessary by default, because the kmesh pods are installed under kmesh-system by default.

But will provider a flag to specify the custom installation namespace for Kmesh in the future.